tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -387,8 +387,11 @@ public function testInvalidAnnotationUsageButIgnoredClass()
         $reader = $this->getReader();
         $ref = new \ReflectionClass('Doctrine\Tests\Common\Annotations\Fixtures\InvalidAnnotationUsageButIgnoredClass');
         $annots = $reader->getClassAnnotations($ref);
-
-        $this->assertEquals(2, count($annots));
+        if ($annots[0] instanceof IgnoreAnnotation) {
+            $this->assertEquals(2, count($annots));
```

Comment:
You realise that this justification is weak right ?
With that kind of logic why not using assertTrue everywhere ?
Why do you think that we need to conform to use only one assertion in the hundred that exists ?

The reason to use assertCount here is because it will make the error message when the test fails an order of magnitude easier to understand and that why we test.